### PR TITLE
Controller

### PR DESCRIPTION
**Pull request by ChrispaixK**
# Blog app - controllers
> Using controllers to handle requests and render empty views
> Understand Rails naming conventions.
> Use params from browser requests in a safe way.

In this PR I : 
* Create user controller
* Create post controller
* Add actions in user controller
* Add action in post controller
* Render empty views with placeholder only
* Fix linters errors

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    @posts = Post.where(user_id: params[@user.id])
+  end
+
+  def display
+    @posts = Post.find(params[:id])
+  end
```
Comment (TedLivist) [inline app/controllers/posts_controller.rb]: - [x] Based on the requirements we should have a `#show` action not `#display`. Kindly fix that. It is also a good idea to exhaust the inbuilt route actions provided by Rails before creating similar ones

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @user = User.all
+  end
+
+  def display
+    @user = User.find(params[:user_id])
+  end
```
Comment (TedLivist) [inline app/controllers/users_controller.rb]: - [x] Based on the requirements we should have a `#show` action not `#display`. Kindly fix that. It is also a good idea to exhaust the inbuilt route actions provided by Rails before creating similar ones

File: app/views/posts/display.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<h2>Here is a list of posts for a given user</h2>
```
Comment (TedLivist) [inline app/views/posts/display.html.erb]: - [x] Kindly rename this file to match the action renamed in the controller: `show.html.erb`

File: app/views/users/display.html.erb [hunk 4]
```diff
@@ -0,0 +1 @@
+<h2>users display<h2>
```
Comment (TedLivist) [inline app/views/users/display.html.erb]: - [x] Kindly rename this file to reflect the action defined in the controller: `show.html.erb`

File: config/routes.rb [hunk 5]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get 'users', to: 'users#index'
+  get '/users/:user_id', to: 'users#display'
+  get 'users/:user_id/posts', to: 'posts#index'
+  get 'users/:users_id/posts/:id', to: 'posts#display'
```
Comment (TedLivist) [inline config/routes.rb]: - [x] There seems to be a lot of inconsistency with your routing here. Sometimes you define `users_id`, other times you define `user_id`. Rails takes care of these routes for us. We can nest these routes and also use the `resources` helper from Rails.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @ChrispaixK,

Good job so far!
✔️ Good documentation
✔️ Definition of index actions
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (ChrispaixK) [reply app/controllers/posts_controller.rb]: Thank you @TedLivist 
Let me fix that 👍🏾

File: config/routes.rb (hunk 5 above)
Comment (ChrispaixK) [reply config/routes.rb]: Thank you for your help @TedLivist 
I appreciate

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @ChrispaixK,

✔️ Good implementation of routes

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**